cluster/src/main/java/com/hartwig/pipeline/tertiary/linx/LinxGermline.java

```diff
@@ -56,16 +63,32 @@ public String namespace() {
     }
 
     @Override
-    public List<BashCommand> commands(final SomaticRunMetadata metadata) {
-        return Collections.singletonList(new LinxCommand(metadata.tumor().sampleName(),
-                gripssGermlineVariantsDownload.getLocalTargetPath(),
-                resourceFiles.version(),
-                VmDirectories.OUTPUT,
-                resourceFiles.lineElements(),
-                resourceFiles.ensemblDataCache(),
-                resourceFiles.driverGenePanel(),
-                resourceFiles.gridssBreakpointPon(),
-                resourceFiles.gridssBreakendPon()));
+    public List<BashCommand> tumorReferenceCommands(final SomaticRunMetadata metadata) { return buildCommand(metadata, TUMOR_REFERENCE); }
+
+    @Override
+    public List<BashCommand> referenceOnlyCommands(final SomaticRunMetadata metadata) { return buildCommand(metadata, REFERENCE_ONLY); }
+
+    @Override
+    public List<BashCommand> tumorOnlyCommands(final SomaticRunMetadata metadata) { return Stage.disabled(); }
+
+    private List<BashCommand> buildCommand(final SomaticRunMetadata metadata, final InputMode inputMode) {
```

Comment:
we could pass the sample name in rather than the metadata and remove the need to check the mode